TechtonicaModLoader/MyClasses/ThunderStore.cs

```diff
@@ -98,7 +98,19 @@ private static async Task<List<ThunderStoreMod>> GetAllThunderStoreMods() {
 
             List<ThunderStoreMod> mods = JsonConvert.DeserializeObject<List<ThunderStoreMod>>(json);
             Log.Debug($"Got {mods.Count} mods from ThunderStore");
-            
+
+            // Mark all mods older than the 1.0 release as deprecated.
+            int preReleaseMods = 0;
+            foreach (ThunderStoreMod thunderStoreMod in mods) {
+                // The DateTime strings from Thunderstore are ISO 8601 compliant so don't need a format provider for Parse.
+                if (DateTime.Parse(thunderStoreMod.date_updated) < ModManager.techtonicaReleaseDate && 
+                    !ModManager.allowedMods.Contains(thunderStoreMod.name)) {
+                    thunderStoreMod.is_deprecated = true;
+                    ++preReleaseMods;
+                }
+            }
+            Log.Debug($"Marked {preReleaseMods} pre-release mods as deprecated");
+
             List<ThunderStoreMod> depricatedMods = mods.Where(mod => mod.is_deprecated).ToList();
             mods = mods.Where(mod => !mod.is_deprecated).ToList();
             mods = mods.Where(mod => mod.name != "r2modman").ToList();
```

Comment:
You can simplify your code by adding another .Where call down here:
`mods = mods.Where(mod => DateTime.Parse(mod.date_updated) >= ModManager.techtonicaReleaseDate).ToList()`

Could also merge those three .Where calls into one with some &&'s

Reply:
Yep, sounds good.    

I habitually shy away from LINQ as it's easy to hide perf issues but it seems fine in this context.

Reply:
Looking at this again, we get to count and display the number of pre-release mods filtered from Thunderstore at the expense of a bit more code.   If you don't really see this as valuable, I can change it.

Reply:
I largely followed what we discussed but condensed it to two LINQ statements.